index.js

```diff
@@ -9,6 +9,9 @@ var read = require('read-all-stream');
 var timeout = require('timed-out');
 var isReadableStream = require('isstream').isReadable;
 
+var httpAgent = new http.Agent({maxSockets: Infinity});
+var httpsAgent = new https.Agent({maxSockets: Infinity});
```

Comment:
Can you add a `TODO` about removing this when Node 0.10 is not supported anymore?